tpl/vagrantfile.tpl

```diff
@@ -0,0 +1,12 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+  config.vm.synced_folder ".", "/vagrant", type: "rsync"
```

Comment:
I think we should either create a generic Vagrant template that will work with all supported providers natively or use this template only for libvirt boxes with help of the `only` directive as described here https://www.packer.io/docs/templates/hcl_templates/blocks/build/post-processor#run-on-specific-builds